e2e_tests/test_homepage.py

```diff
@@ -3,7 +3,7 @@
 
 
 def test_has_title(page: Page):
-    page.goto("http://localhost:5000/")
+    page.goto("/")
```

Comment:
need to decide how we will run playwright tests in ci. I think it makes sense to run them against the sandbox deployment. Im going to do a separate PR to add a deploy to sandbox workflow on merge to maim, after which we can run the playwright tests against it!